linode/manager
# refactor: [M3-6356] - MUI v5 Migration - `Components > LineGraph`

## Description 📝
Refactor `LineGraph` with `styled` api

## Major Changes 🔄
- Removed `jss` and `makeStyles` for `styled` api
- Updated imports/exports
- Fixed bug for `LinodeEntityDetail.tsx` causing console error `Cannot have a div within a p tag`
- Uncovered bug for `NetworkGraphs.tsx` where the graphs weren't spanning full width after properly refactoring (probably was masked by some collision between `jss` and new styles`

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-06-16 at 2 13 35 PM](https://github.com/linode/manager/assets/125309814/dbbed174-0d56-4003-9f6f-22f76d7d428b) | ![Screenshot 2023-06-16 at 2 14 19 PM](https://github.com/linode/manager/assets/125309814/26809965-552f-48c9-b6b9-cc24c675a864) |

## How to test 🧪
1. Go to a linode detail page where you see line graphs
2. You shouldn't notice any observable changes with exception of what I fixed
3. Check different breakpoints
4. Toggle legends (buttons) and ensure they're crossed out
